README.md

```diff
@@ -167,6 +167,32 @@ $bunny = new Client($configuration);
 $bunny->connect();
 ```
 
+### Always run moving parts in a fiber
+
+The major different between Bunny v0.5 and v0.6 is that it's been rebuild using [`Fibers`](https://reactphp.org/async/). This means that every method on `Client` and `Channel` must be called inside a fiber. In the examples this will be called using a `Loop::futureTick` call. In your applications this might be some other trigger like a HTTP request. As long as somewhere in the direct call stack this call is inside a fiber you'll be good.
```

Comment:
> using [`Fibers`](https://reactphp.org/async/).

Does Bunny Bunny require using ReactPHP's implementation of Fiber context?
If not - I suggest hinting to PHP docs at first, and to hint about the recommended abstraction - ReactPHP's

Reply:
It's using the ReactPHP implementation